packages/react-vapor/src/components/browserPreview/BrowserPreview.tsx

```diff
@@ -6,19 +6,23 @@ import {Tooltip} from '../tooltip';
 
 export interface BrowserPreviewProps {
     headerDescription?: string;
+    title?: string;
 }
 
-export const BrowserPreview: React.FunctionComponent<BrowserPreviewProps> = ({children, headerDescription}) => (
+export const BrowserPreview: React.FunctionComponent<BrowserPreviewProps> = ({children, title, headerDescription}) => (
     <div className="browser-preview flex flex-column">
-        <BrowserPreviewHeader tooltipTitle={headerDescription ?? DefaultHeaderDescription} />
+        <BrowserPreviewHeader tooltipTitle={headerDescription ?? DefaultHeaderDescription} title={title ?? 'Preview'} />
```

Comment:
Add `Preview` in local file please

Reply:
We do not have file for strings in react-vapor, but I've added it in a const inside the component (as soon as a prop for title is provided, it will be overriden)